packages/code-du-travail-frontend/server/server.js

```diff
@@ -59,6 +59,7 @@ nextApp.prepare().then(() => {
         directives: {
           defaultSrc: [
             "'self'",
+            "http://127.0.0.1:1337",
```

Comment:
this should only added in devMode

Reply:
The issue is that the frontend has to be launched in production mode hence the obligation to put this rule here. Maybe an env var ? I tried to make the test suit work in dev mode but maybe I should try harder

Reply:
Now works on dev mode !